_Week9/Group5/README.md

```diff
@@ -0,0 +1,43 @@
+
+We define the true average treatment effect (ATE) as the effect of attending charters on test scores across all 5,000 children in our sample. The true ATE is simply the mean of the individual treatment effects (ITE) across all children (i.e., the difference between each child's test score attending charter and his/her test score attending public). It is mathematically equivalent to say the true ATE = mean(ts_charter) - mean(ts_public). We estimate the biased ATE using an IV regression of test scores on a dummy variable for attending a charter, where admission to a charter serves as an instrument. The regression coefficient represents the ATE across all students *who actually attend charters*, i.e., students who were admitted *and* who choose to attend (because their test scores would be higher attending charters than public).
+
+Why does the biased ATE differ from the true ATE? The bias is introduced by students who are admitted to charters but choose not to attend, as they would have lower test scores attending charters. As a result, the biased ATE is an overestimate of the true ATE. The latter reflects the impact of charter attendance across *all students*, whereas the biased ATE reflects the impact on the *compliers amongst the treated*, i.e., the admitted who *choose* to attend charters.
+
+Why does this discussion matter? The biased ATE reflects a model specification pioneered by Angrist, Imbens, and Rubin (1996) that has become common the charter-school literature. However, it is imperative to underline how the bias in the ATE changes the appropriate takeaways from a study using this model. Namely, a sufficiently large and significant biased ATE should only serve as evidence for providing charters as an alternative *choice* to public schools. Such a biased ATE cannot, in good faith, be used to argue for *replacing* all public schools with charters.
+
+## power-calculation.do
+We loop through our simulation 100 times. (We will do more iterations for our final project, when we have more time.) For each iteration, we add a row to a matrix with columns for true ATE, biased ATE, the upper and lower bounds of the 95% confidence interval of the biased ATE, the mean test scores for public and private, and the standard deviations in test scores for public and private. We then imported these data into Stata's memory and cleaned up the variables.
+
+Next, for each iteration (i.e., row), we calculate the minimum sample size at a 5% significance level and 0.8 power. These results are imputed into a new variable.
+
```

Comment:
Do you think it would be useful to briefly explain why we are using 0.8 power?

Reply:
Added 0.8 power appears standard in the literature.